ci-infra/ami-build/scripts/detach-self.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
This is not a script I've evaluated. It was imported wholesale, and likely needs editing